graphics/openvdb/Portfile

```diff
@@ -7,28 +7,45 @@ PortGroup               active_variants 1.1
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               boost 1.0
 
-github.setup            AcademySoftwareFoundation openvdb 7.2.3 v
-revision                1
-homepage                https://www.openvdb.org/
+github.setup            AcademySoftwareFoundation openvdb 8.0.0 v
 categories              graphics
-license                 {MPL-2 LGPL-2.1+}
-maintainers             {mcalhoun @MarcusCalhoun-Lopez} openmaintainer
-description             sparse volume data structure and tools
-long_description        OpenVDB is an open source C++ library comprising a novel hierarchical data structure \
-                        and a large suite of tools for the efficient storage and manipulation of \
-                        sparse volumetric data discretized on three-dimensional grids.
 platforms               darwin
+license                 {MPL-2 LGPL-2.1+}
+maintainers             {mcalhoun @MarcusCalhoun-Lopez} \
+                        @jasonliu-- openmaintainer
 
-checksums               rmd160  b85de208ed29ab1675956f78fd867eeaaecc591d \
-                        sha256  1e929a7614e7b47d9b51c56c5e76c89cb3fef572d6a6affb89b45af2c81a297c \
-                        size    2289366
+homepage                https://www.openvdb.org/
+description             sparse volumetric data structure and tools
+long_description        OpenVDB is an open source C++ library \
+                        comprising a novel hierarchical data structure \
+                        and a large suite of tools for the efficient \
+                        storage and manipulation of sparse volumetric \
+                        data discretized on three-dimensional grids. \
+                        It was developed by DreamWorks Animation for \
+                        use in volumetric applications typically \
+                        encountered in computer-generated graphics and \
+                        animation.
+
+checksums               rmd160  68e677f58b7fea6dc4bd8e6d55c531205195625c \
+                        sha256  28a811bae32b62a2080aad4776c91d4453a7d52e2edbe1ab4a21783ec4aaa6bb \
+                        size    2287123
 
 compiler.cxx_standard   2014
-# error: cannot refer to variable template 'Contains' without a template argument list
+# error: cannot refer to variable template 'Contains' without a template
+# argument list
 #     static_assert(NodeChainType::template Contains<NonConstNodeType>,
 #                   ~~~~~~~~~~~~~~~         ^
 compiler.blacklist-append {clang < 800}
 
+depends_build-append    port:pkgconfig
+
+depends_lib-append      port:zlib \
+                        port:blosc \
+                        port:tbb \
+                        path:lib/pkgconfig/glfw3.pc:glfw \
+                        port:ilmbase \
+                        port:openexr
+
```

Comment:
Nitpick: Might be better to leave the patchfiles where they were before, prior to the dependencies, as the change makes it more difficult to discern the tangible changes in this PR.

Reply:
I tend to like to place the patchfiles in line with the MacPorts phases, i.e. `pre-patch`, `patchfiles`, `post-patch`, `pre-configure`, etc., but I see your point, relocating the patchfiles does cause the diff to become quite messy. So I've restored their original location, as suggested.